@jules lint:
```
run golangci-lint
  Running [/home/runner/golangci-lint-2.11.4-linux-amd64/golangci-lint config path] in [/home/runner/work/g2/g2] ...
  Running [/home/runner/golangci-lint-2.11.4-linux-amd64/golangci-lint run] in [/home/runner/work/g2/g2] ...
  Error: cmd/g2/site_serve.go:179:2: ineffectual assignment to aggUseFlags (ineffassign)
  	aggUseFlags := make(map[string]*AggUseFlag)
  	^
  1 issues:
  * ineffassign: 1
```

--- a/cmd/g2/site_serve.go
+++ b/cmd/g2/site_serve.go
@@ -176,7 +176,6 @@ func newSiteServer(sites []*SiteData) (*SiteServer, error) {
 	aggPackages := make(map[string]*AggPackage)
 	aggLicenses := make(map[string]*AggLicense)
 	aggProjects := make(map[string]*AggProject)
-	aggUseFlags := make(map[string]*AggUseFlag)
 
 	for _, site := range sites {
 		if site.Projects != nil {

